tests/unit/test_utils.py

```
@@ -346,6 +346,71 @@ async def call_from_async_context() -> None:
+async def test_aclose_quietly_prefers_aclose_over_close() -> None:
+    """An async generator style aclose() wins when both spellings exist."""
+    stream = _Closable(aclose=True, close=True)
+
+    await aclose_quietly(stream)
+
+    assert stream.calls == ["aclose"]
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("sync", [True, False])
+async def test_aclose_quietly_calls_close_sync_or_async(sync: bool) -> None:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)